types/tipset_key.go

```diff
@@ -128,6 +131,17 @@ func (s TipSetKey) String() string {
 	return out + " }"
 }
 
+// Serialize returns a serialize string of TipSetKey use gob and base64.
```

Comment:
Blocking: We already have a canonical encoding for `TipSetKey` into cbor defined at the top of this file.  I don't see the value in defining a gob serialization like this right now.  Perhaps I'm missing something so feel free to explain why you think this is important.

Reply:
OK, Remove it. and the method just for review to if it need.
https://filecoinproject.slack.com/archives/CEHHJNJS3/p1564377470109500